fastlane/Fastfile

```diff
@@ -51,6 +51,7 @@ lane :automatic_bump do |options|
     github_rate_limit: options[:github_rate_limit]
   )
   options[:next_version] = next_version
+  UI.user_error!('Skipping automatic bump since the next version doesn\'t include public facing changes') if type_of_bump == :skip
```

Comment:
Hmm we will get an error on CI if we do it like this... I think we should just log this normally and finish (without executing the bump) instead. The same can be said for the next line, we probably should just log it and finish without errors. I think otherwise we will get a lot of unnecessary errors in CI (even if they are almost invisible, since they are generated automatically).